taxi/views.py

```diff
@@ -87,3 +85,43 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreateForm
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverLicenseUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    form_class = DriverLicenseUpdateForm
+    template_name = "taxi/driver_update_license.html"
+
+
+@login_required
+def car_detail_view(request, pk):
+    driver = Driver.objects.get(id=request.session["_auth_user_id"])
+    car = Car.objects.get(pk=pk)
+    is_driver = driver in car.drivers.all()
+    context = {
+        "car": car,
+        "is_driver": is_driver
+    }
+    return render(request, "taxi/car_detail.html", context=context)
+
+
+@login_required
+def car_update_drivers(request, pk):
```

Comment:
Write as class

Reply:
@ArsenPidhoretskyi I've changed car detail view to class-based. But I don't understand how to implement this assign\delete driver feature without a function-based view.